dimension/helpers.go

```diff
@@ -61,7 +62,7 @@ func handleDimensionErr(ctx context.Context, w http.ResponseWriter, err error, d
 			status = http.StatusConflict
 		default:
 			status = http.StatusInternalServerError
-			err = errs.ErrInternalServer
+			err = errors.WithMessage(err, errs.ErrInternalServer.Error())
```

Comment:
Could just use fmt.Errorf()? Also would probably be better to format it slightly, looks like it will be two strings concatenated with no spaces atm

Reply:
(example: fmt.Errorf("%s: %s", errs.InternalServer, err)

Reply:
https://play.golang.org/p/8A8Qkh00Bde example of how it will work.

Reply:
I'd prefer to use this WithMessage method as it's specifically designed for this purpose. It also retains the original error, using format would just take the original error string but the full struct would be lost.